apps/webapp/mix.exs

```diff
@@ -0,0 +1,85 @@
+defmodule Webapp.Mixfile do
+  use Mix.Project
+
+  def project do
```

Comment:
Shouldn't we need to add **:build_path**, **:config_path**, **:deps_path**, and **:lockfile** keys in `project`? 

>“You can see the configurations for :build_path, :config_path, :deps_path, and :lockfile. That’s all it takes to make something an umbrella child. ”
>Excerpt From: Chris McCord, Bruce Tate, and José Valim. “Programming Phoenix (for Hisa Ishibashi ).” iBooks.